tests/TestCase.php

```diff
@@ -2,6 +2,34 @@
 
 namespace ipl\Tests\Web;
 
+use ipl\Html\ValidHtml;
+use PHPUnit\Util\Xml;
+
+// phpcs:disable
+if (class_exists('PHPUnit_Util_XML')) {
+    // Support older PHPUnit versions
+    class_alias('PHPUnit_Util_XML', 'PHPUnit\\Util\\Xml');
+}
+
```

Comment:
Isn't the phpcs:enable instruction required?

Reply:
The test seems to work with phpcs:enable. I removed the whole line, since phpcs:enable should be set by default.
This is also from the Original method in ipl-html. Maybe it caused Problems with older PHP versions.